tools/modules/meterpreter_reverse.erb

```diff
@@ -38,7 +34,7 @@ module MetasploitModule
     opts = {
```

Comment:
Can we add a comment above the `def generate` with a comment like:
```
# Module generated by tools/modules/generate_mettle_payloads.rb
```

So that future folk know not to modify the modules manually 👀

Reply:
for sure